lambda-views/macros/lambda/lambda_union.sql

```diff
@@ -0,0 +1,37 @@
+{% macro lambda_union(historical_relation, model_sql) %}
+
+{% set unique_key = config.get('unique_key', none) %}
+
+with historical as (
+    
+    select * from {{ historical_relation }}
```

Comment:
Would love to see:
```
select
    *,
    'view' as _dbt_lambda_view_source,
    '{{ run_started_at}}' as _dbt_last_run_at
from {{ historical_relation }}
```
I suspect that using these might result in some weird errors. Capturing these things up front will make it soooo much easier to debug things later